components/bulletins/BulletinCardCollection.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+    <div class="flex flex-col w-full">
+        <div
+            class="bulletinCardCollection__cardContainer justify-center md:justify-start"
+        >
+            <!-- <slot></slot> -->
+            <bulletin-card
+                v-for="(bulletin, i) in $t('bulletins')"
+                :key="`index_bulletin_${i}`"
+                :title="bulletin.title"
+                :description="bulletin.description"
+                :linkStr="bulletin.linkStr"
+                :linkUrl="bulletin.linkUrl"
+            >
+            </bulletin-card>
+        </div>
+    </div>
+</template>
+
+<script>
+import i18n from './BulletinCard.i18n'
+export default {
+    i18n,
+    name: 'BulletinCardCollection',
+}
+</script>
+
+<style lang="postcss" scoped>
+.bulletinCardCollection__cardContainer {
+    @apply flex flex-wrap justify-center;
+}
+
+.bulletinCardCollection__cardContainer > .bulletinCard {
+    margin: 0 0 60px 0;
+    padding: 5px;
+    @media (min-width: 415px) {
+        margin: 30px 50px 40px 5px;
+    }
+}
+
+.bulletinCardCollection__cardContainer > .bulletinCard:hover {
+    @apply cursor-pointer box-content;
+    margin: 25px 45px 35px 0px;
```

Comment:
Only `@apply cursor-pointer;` I think, from my understanding of Figma's spec. Unlike sponsor cards, hover to enlarge the card, the bulletin cards should hover to change the colors.

I think in ZH, the card's width and height look great. However, in EN, the sentences are so   long that they break the 4-liner design. Let's ask the designer to see how to deal with the issue.